charts/stable/nextcloud/templates/_configmap.tpl

```diff
@@ -0,0 +1,363 @@
+    {{- end }}
+
+    {{/* Expirations */}}
+    NX_ACTIVITY_EXPIRE_DAYS: {{ .Values.nextcloud.expirations.activity_expire_days | quote }}
+    NX_TRASH_RETENTION: {{ .Values.nextcloud.expirations.trash_retention_obligation | quote }}
+    NX_VERSIONS_RETENTION: {{ .Values.nextcloud.expirations.versions_retention_obligation | quote }}
+
+    {{/* General */}}
+    NX_RUN_OPTIMIZE: {{ .Values.nextcloud.general.run_optimize | quote }}
+    NX_DEFAULT_PHONE_REGION: {{ .Values.nextcloud.general.default_phone_region | quote }}
+
+    {{/* Files */}}
+    NX_SHARED_FOLDER_NAME: {{ .Values.nextcloud.files.shared_folder_name | quote }}
+    NX_MAX_CHUNKSIZE: {{ .Values.nextcloud.files.max_chunk_size | quote }}
+
+    {{/* Logging */}}
+    NX_LOG_LEVEL: {{ .Values.nextcloud.logging.log_level | quote }}
+    NX_LOG_FILE: {{ .Values.nextcloud.logging.log_file | quote }}
+    NX_LOG_FILE_AUDIT: {{ .Values.nextcloud.logging.log_audit_file | quote }}
+    NX_LOG_DATE_FORMAT: {{ .Values.nextcloud.logging.log_date_format | quote }}
+    NX_LOG_TIMEZONE: {{ .Values.TZ | quote }}
+
+    {{/* ClamAV */}}
+    NX_CLAMAV: {{ .Values.nextcloud.clamav.enabled | quote }}
+    {{- if .Values.nextcloud.clamav.enabled }}
+    NX_CLAMAV_HOST: {{ printf "%v-clamav" (include "tc.v1.common.lib.chart.names.fullname" $) }}
+    NX_CLAMAV_PORT: {{ .Values.service.clamav.ports.clamav.targetPort | quote }}
+    NX_CLAMAV_STREAM_MAX_LENGTH: {{ .Values.nextcloud.clamav.stream_max_length | quote }}
+    NX_CLAMAV_FILE_MAX_SIZE: {{ .Values.nextcloud.clamav.file_max_size | quote }}
+    NX_CLAMAV_INFECTED_ACTION: {{ .Values.nextcloud.clamav.infected_action | quote }}
+    {{- end }}
+
+    {{/* Collabora */}}
+    NX_COLLABORA: {{ .Values.nextcloud.collabora.enabled | quote }}
+    {{- if .Values.nextcloud.collabora.enabled }}
+    # TODO: Update to ingress name
+    NX_COLLABORA_URL:
+    NX_COLLABORA_ALLOWLIST: {{ join "," .Values.nextcloud.collabora.allow_list | quote }}
+    {{- end }}
+
+    {{/* URLs TODO: */}}
+    NX_OVERWRITE_HOST: {{ .Values.chartContext.APPURL | quote }}
+    NX_OVERWRITE_PROTOCOL: https
+    NX_OVERWRITE_CLI_URL: {{ .Values.chartContext.APPURL | quote }}
+    # TODO: Verify what is needed
+    NX_TRUSTED_DOMAINS: |
+      127.0.0.1
+      {{ $hosts | default "TODO:fix" }}
+      kube.internal.healthcheck
+      {{ printf "%s-nextcloud" .Release.Name }}
+      {{ printf "%v-nextcloud-backend" .Release.Name }}
+      {{ printf "%v-nextcloud:%v" .Release.Name .Values.service.main.ports.main.port }}
+      {{ printf "127.0.0.1:%v" .Values.service.main.ports.main.port }}
+      {{ .Values.nextcloud.AccessIP | default "localhost" }}
+    # TODO: Verify what is needed
+    NX_TRUSTED_PROXIES: |
+      localhost
+      127.0.0.1
+      {{ .Values.chartContext.podCIDR }}
+      {{ .Values.chartContext.svcCIDR }}
```

Comment:
Yeah so above trusted proxies do not have to contain 127.0.0.1 or the default scale cluster cidr.

Reply:
svc cidr should be enough I think, right?